### ISSUE TITLE
Mini project 주제 선정 및 역할분담 이야기해요!

### ISSUE THREAD

**Issue by seungmoonchoi1** [OSSL2023csmksh/teamproject#2]

**Comment by seungmoonchoi1** [comment]
프로젝트 주제는 전에 이야기 했던 것처럼 야구 선수 스탯 검색기로 하려고 하는데 괜찮나요?

**Comment by rescue-possible** [comment]
네 좋은 것 같아요 타자들의 각 스탯별로 순위를 나열해주는 기능이 있으면 좋을 것 같아요
중요한 지표가 홈런, 타율, 타점 이 세가지가 있으니까 이것을 중심으로 기능을 만들어 보는게 어떨까요?

**Comment by seungmoonchoi1** [comment]
네 좋습니다!! 마지막 기능으로는 가중치를 둔 스텟을 종합적으로 계산을 해서(승리기여도와 비슷한 느낌으로) 예상 mvp 순위 별로 나열하는 기능을 넣는 것은 어떨까요?

**Comment by rescue-possible** [comment]
재밌을것같아요

**Comment by seungmoonchoi1** [comment]
역할 분담을 어떻게 할까요?

**Comment by rescue-possible** [comment]
wiki랑 코드구현 중심으로 역할을 나눠볼까요

**Comment by seungmoonchoi1** [comment]
넵 말씀 하신대로 역할 분담 나눠 봤습니다!
코드
1. CRUD+SAVE+LOAD 김승혁
2. main 함수 + 나머지 최승문
WIKI
1. 김승혁 : mini project의 주제 및 소개
2. 최승문 : mini project의 기능 설명

**Comment by rescue-possible** [comment]
네 일단 이렇게 진행하면 될 것 같아요!

**Comment by rescue-possible** [comment]
일단 오늘은 과제내용대로 구체적인 코드를 만들기보다 기획과 설계과정만 하면 될 것 같아요

**Comment by seungmoonchoi1** [comment]
네 WIKI 제가 맡은 부분 작성 했습니다!

**Comment by rescue-possible** [comment]
wiki에 주제 올렸습니다!

**Comment by seungmoonchoi1** [comment]
저희 구현할 프로젝트의 구조체 정의 및 함수 정의 개발 한 내용 올리겠습니다.

typedef struct {
  char name[20];
  char position[20];
  float AVG;
  int RBI;
  int HR;
  float prmvp; // 예상 mvp 순위를 위한 세이버 매트릭스 값
} Stat;

int selectMenu();
int addStat(Stat *s); // 선수의 스탯을 추가하는 함수
void readStat(Stat s); // 선수의 스탯 출력하는 함수
int updateStat(Stat *s); // 선수의 스탯 수정하는 함수
void listStat(Stat *s[], int count); // 전체 등록된 선수 스탯 출력
int selectDataNo(Stat *s[], int count); // 선수 번호 불러오는 함수
int deleteStat(Stat *s); // 등록된 선수 삭제하는 함수
void saveData(Stat *s[], int count); // 데이터를 파일에 저장하는 함수
void searchName(Stat *s[], int count); // 선수 이름검색
void listStatByHR(Stat *s[], int count); // 선수 홈런순위 sorting 함수
void listStatByAVG(Stat *s[], float count); // 선수 타율순위 sorting 함수
void listStatByRBI(Stat *s[], int count); // 선수 타점순위 sorting 함수
int selctDataStat(Stat *s[], int count); // 타율,홈런, 타점 중 순위를 보기 원하는 스탯 고르게 하는 함수
int prMVP(Stat *s[], int count); // 예상 MVP 순위 계산하는 함수
void listprMVP(Stat *s[], int count); // 예상 MVP 순위 나열하는 함수
int loadData(Stat *s[]); // 저장된 데이터 불러오는 함수

**Comment by rescue-possible** [comment]
참조하여 Wiki란에 만들겠습니다!
